module/dialogs/FightDialog.ts

```diff
@@ -111,19 +111,19 @@ export class FightDialog extends ExtendedTestDialog<FightDialogData> {
             this._toggleHidden(e.target);
         });
         html.find('.fighters-grid input, .fighters-grid select').on('change', (e: JQuery.ChangeEvent) => this.updateCollection(e, this.data.data.participants));
-
-        html.find('button[data-action="rollSpeed"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "speed"); });
-        html.find('button[data-action="rollPower"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "power"); });
-        html.find('button[data-action="rollAgility"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "agility"); });
-        html.find('button[data-action="rollSkill"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "skill"); });
-
-        html.find('img[data-action="openSheet"]').on('click', (e: JQuery.ClickEvent) => {
-            const id = e.target.dataset.actorId || "";
-            game.actors.find(a => a._id === id).sheet.render(true);
+        ["Speed", "Power", "Agility", "Skill", "Steel"].forEach((attr: string) => {
+            html.find('button[data-action="roll'+attr+'"]')
+                .on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, attr.toLowerCase() as constants.FightAttr); });
         });
+        const openSheet = (e: JQuery.ClickEvent) => {
+            const id = e.currentTarget.attributes.getNamedItem("data-actor-id").nodeValue || "";
+            game.actors.find(a => a._id === id).sheet.render(true);
+        };
+        html.find('img[data-action="openSheet"]').on('click', openSheet);
+        html.find('div[data-action="openSheet"]').on('click', openSheet);
     }
     
-    private _handleRoll(e: JQuery.ClickEvent, type: "speed" | "agility" | "power" | "skill") {
+    private _handleRoll(e: JQuery.ClickEvent, type: constants.FightAttr) {
```

Comment:
I think it's probably fine to pull `FightAttr` from `constants.ts` into this file. I don't foresee anyone outside of the fight dialog needing to reference it other than the one use in `FightRoll.ts`. Open to thoughts on this though

Reply:
May be better to keep in the general list as if any further refactoring occurs(fight gets split into multiple files for whatever reason) it might get forgotten. Don't mind either way though - very minor thing

Reply:
I'll leave this entirely up to you, but I think it's better to keep only broadly used constants in `constants.ts` so it doesn't become overwhelmed with bits from every single file that wants to define something. I would expect if fight gets split into multiple files somehow, there would still be some logical place to keep constants that are mostly related to the fight dialog itself.